### DISCUSSION TITLE
3Q0907530Q problem (Audi S3 8V)

### DISCUSSION THREAD

**Discussion by paratkd** [jilleb/MQB-FPA#103] [Q&A]
Hi all. I'm stuck.

Vehicle: Audi S3 8V.5 (2018)

Upgraded my gw software from 3q0907530L (4326) to Q (4343). Unfortunately, reflashing my Odis OEM dataset XML doesn't work, neither does ANY other dataset I've tried for similar vehicles. I have a fault logged with "dataset implausible" and my dash is lit up like christmas with various things not working (TCS/ECS, cruise, auto hold, hill hold, no driving mode selection etc).

I can't even revert back to software L as the vag flash file doesn't exist (superseded perhaps?). I've also tried AA, G and C. No dice!

Only option I can see is to physically replace the gateway module [perhaps from my current older Lear (4xxx), to a continental (5xxx?)]. But I'd hope to avoid that. Any ideas/suggestions greatly appreciated!

Many thanks 🙏🏻

**Comment by przebiegly0** [comment]
the L firmware you can find on mibsolution.one or here is the direct link from my resources > http://vag.netzloty.pl/Firmware/09/FL_3Q0907530L_4326__V001_S.odx

or maybe flash with AC firmware and check this dataset from audi a3 > https://github.com/jilleb/MQB-FPA/files/8609989/3Q0907530AC.zip
